scripts/sysman/EXT_RasState.rst

```diff
@@ -0,0 +1,61 @@
+<%
+import re
+from templates import helper as th
+%><%
+    OneApi=tags['$OneApi']
+    s=tags['$s']
+    S=s.upper()
+%>
+:orphan:
+
+.. _ZES_extension_ras_state:
+
+========================
+ Ras Get State Extension
+========================
+
+API
+----
+
+* Functions
+
+    * ${s}RasGetStateExt
+
+* Structures
+
+    * ${s}_ras_state_ext_t
+
+
+Ras State
```

Comment:
@wdamon-intel  @jandres742  added this section to document intentions for extension and also some pseudo code. please check